tracing-mock/src/collector.rs

```diff
@@ -311,7 +313,7 @@ where
     ///
     /// [`Collect::drop_span`]: fn@tracing::Collect::drop_span
     /// [`close_span`]: fn@Self::close_span
-    #[deprecated(since = "0.1.0", note = "use `MockCollector::close_span` instead")]
```

Comment:
why was the deprecated attribute on this function removed?

Reply:
Adding the deprecated attribute meant adding `#[allow_deprecated]` on every single usage of it throughout the tracing tests. I felt it better to leave the deprecated attribute off until I can add a replacement. Then the current usage of `drop_span` can be replaced and this method can be marked as deprecated.